schema.graphql

```diff
@@ -22,3 +22,31 @@ type Payment @entity {
   feeAmountInCrypto: BigDecimal
   maxRateTimespan: Int
 }
+
+type EscrowEvent @entity {
+  id: ID!
+  contractAddress: Bytes!
+  reference: Bytes!
+  block: Int!
+  timestamp: Int!
+  txHash: Bytes!
+  eventType: String!
+  from: Bytes!
+  gasUsed: BigInt!
+  gasPrice: BigInt!
+}
+
+type Escrow @entity {
```

Comment:
about my comment on relationship between Escrow and Events, here is the [doc](https://thegraph.com/docs/developer/create-subgraph-hosted#one-to-many-relationships)